LibOS/shim/src/sys/shim_brk.c

```diff
@@ -98,16 +98,25 @@ int init_brk_region (void * brk_region)
 
             brk_region = vma.addr + vma.length;
         }
+
+        /*
+         * Create the bookkeeping before allocating the brk region.
+         * The bookkeeping should never fail because we already confirm

```

Comment:
`we already confirm` -> `we've already confirmed`

Reply:
Fixed.